sympy/physics/control/lti.py

```diff
@@ -0,0 +1,181 @@
+from sympy import Basic, Mul, degree, Symbol, expand, cancel, Expr
+from sympy.core.numbers import Integer, Float
+from sympy.core.sympify import sympify, _sympify
+
+__all__ = ['TransferFunction',]
+
+
+class TransferFunction(Basic):
+
+    def __new__(cls, num, den, var):
+        num, den = _sympify(num), _sympify(den)
+
+        if not isinstance(var, Symbol):
+            raise TypeError("Variable input must be a Symbol.")
+        if den == 0:
+            raise ValueError("TransferFunction can't have a zero denominator.")
+
+        if (isinstance(num, Expr) and num.has(Symbol)) or isinstance(num, Integer):
+            if (isinstance(num, Expr) and num.has(Symbol)) or isinstance(num, Integer):
+                obj = super(TransferFunction, cls).__new__(cls, num, den, var)
+                obj._num = num
+                obj._den = den
+                obj._var = var
+                return obj
+        else:
+            raise ValueError("Unsupported type for numerator or denominator of TransferFunction.")
+
+    @property
+    def num(self):
+        return self._num
+
+    @property
+    def den(self):
+        return self._den
+
+    @property
+    def var(self):
+        return self._var
+
+    def simplify(self):
+        tf = cancel(Mul(self.num, 1/self.den)).as_numer_denom()
```

Comment:
Not working!

Reply:
You should define the `_eval_simplify` method rather than `simplify`. That way `simplify(obj)` works.  The `simplify` method is defined on `Basic` and does not need to be redefined in subclasses.